server/src/domain/game.rs

```diff
@@ -17,10 +17,10 @@ pub struct Game {
 /// The parameters for this should be
 /// all the things needed to make
 /// a new game, but nothing else
-pub fn init(name: String) -> Game {
-    return Game {
-        name,
+pub fn init(name: &str) -> Game {
+    Game {
+        name: name.to_string(),
         players: Vec::new(),
         round: 0,
-    };
```

Comment:
Running `cargo clippy` will give formatting recommendations, and it told me to get rid of `return`.